modules/environment-roles/templates/shared_terraform_policy_2.json.tpl

```diff
@@ -2,123 +2,123 @@
   "Version": "2012-10-17",
   "Statement": [
     {
-      "Sid": "ssmglobal",
+      "Sid": "iam",
```

Comment:
Is it worth renaming this policy to something more specific given it just contains the `iam` permissions: `shared_iam_terraform_policy` ?

Reply:
Yeah makes sense. I've renamed the file and the resources that reference it and I've changed the policy name to TDRSharedIamTerraform$environment